code/modules/economy/persistent_bank_purchases.dm

```diff
@@ -455,6 +456,21 @@ var/global/list/persistent_bank_purchaseables =	list(\
 				return 1
 			return 0
 
+	hellfire
+		name = "Pyromaniac"
+		cost = 1000
+		icon = 'icons/misc/critter.dmi'
+		icon_state = "fire_elemental"
+		icon_dir = SOUTH
+
+		Create(mob/living/M)
+			if(ishuman(M))
+				SPAWN(6 SECONDS)
+					M.set_burning(rand(60,120))
```

Comment:
Preemptive response to `rand(1 MINUTE, 2 MINUTES)`: That breaks the math, since M.set_burning already applies the `SECOND` macro on the input, so it has to be straight numbers.